ButtonA: 0
ButtonB: 1
ButtonX: 3
ButtonY: 4
ButtonLB: 6
ButtonRB: 7
ButtonSelect: 11
ButtonStart: 11
ButtonLeftThumb: 13
ButtonRightThumb: 14
AxisLeftThumbX: 3
AxisLeftThumbY: 2
AxisRightThumbX: 0
AxisRightThumbY: 1
AxisLeftTrigger: 5
AxisRightTrigger: 4
AxisDPadX: 6
AxisDPadY: 7
